Sprint-1/errors/3.js

```diff
@@ -1,6 +1,6 @@
 const cardNumber = 4533787178994213;
-cardNumber = cardNumber.toString();
-const last4Digits = cardNumber.slice(-4);
+const cardNumberStrings = cardNumber.toString();
```

Comment:
Method is different from what I would've done, but ultimately gets the job done. Good work.

Reply:
What other method were you suggesting that I could use, I believe that it could help knowing as well.

Reply:
const cardNumber = 4533787178994213;
const last4Digits = cardNumber.toString().slice(-4);

Reply:
Oh this way is much better since it reduces the lines of code, thanks for that. I didn't think of it.